templates/gateways.html

```diff
@@ -3,21 +3,95 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
 
     <script type="text/javascript" src="{{ url_for('static', filename = 'js/gateway_addr_generation.js') }}"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename = 'js/details.js') }}"></script>
+
 {% endblock %}
 {% block content %}
+    
+
+<div class="container-fluid">
+ 
+<!--
+ <div class="col-9">
+ <i class="fas fa-wallet"></i>
+    <h5>MY WALLET ADDRESS</h5><h2><span class="badge badge-secondary">{{ address }}</span></h2></div>
+-->
+
+
+ <form id="send_form">
+
+<div class="row">GATEWAY: $TN</h5></span></div>
+
+  <div class="col-md-4 offset-md-4"><span><h5>Tx fees: <input required id="tn_fee" maxlength="35"
+                               minlength="35" value="{{ (2000000 + extra_fees)/100000000 }}"> $TN</h5></span></div>
+ </div>
+
+
+<div class="row">
+
+    <div class="col-7 col-md-4">
+     <p>Gateway fees: 0.09 $TN</p>
+            <h2>To Send $TN to Turtle Network, using Waves Network wallet:</h2>
+            <p>Send amount bigger then 0.09 $TN to 3PJ33ZasLyhLaFuK4gHGTFSQbpFBP5Entor with
+                attachment: {{ address }}</p>
+            <h2>To Send $TN to Waves Network, using Turtle Network wallet:</h2>
+            <p>Amount of $TN to send:<input required id="tn_amount" type="number" min="0.01">
+            </div>
+
+
+
+
+    <div class="col-md-4 offset-md-4"><span><h5>EXTRA FEES: {{ extra_fees }}</h5></span></div>
+  </div>
+
+
+  
+    <div class="col-7 col-md-4">
+      <input type="number" class="form-control" placeholder="Token Amount" required id="send_amount_tn" type="number" max="{{ balance }}">
+    </div>
+    <div class="col-7 col-md-4">
+      <input required id="send_addr_tn" maxlength="35" minlength="35" class="form-control" placeholder="Destination Address">
+    </div>
+    
+
+   <!-- 
+
+    <div class="row">
+    <div class="col-7 col-md-4">
+  <button id="send_tn" class="btn btn-outline-dark btn-lg" type="button">SEND$TN</button>
+  </div>
+    <div class="col-md-4 offset-md-4"><span><p style="font-size:12px">When your account is scripted, extra fees will need to be paid. When this isn't scripted, nothing will be added.</p></span></div>
+  </div>
+</form>      
+    </div>
+   -->
+<!--    ----------------------------------------------------------------------------------------------------------------------------> 
+
+<br></br>
+
+
+
+
+<!--
```

Comment:
Why the deuplicated code between comments?

Reply:
I'm still working on that. working of laptop switching between screens is not working nicely. this <i> <em> error is just bs ....Google says <i> sonarcloud says <em> I will do less PR so my workflow doesn’t interfere

Reply:
trust in sonarcloud, it's a lot used in IT-land I would say

Reply:
well the exception is for icons sonar cloud just see don't see it ...but its ok i changed it